eslint-yoshi-base.js

```diff
@@ -0,0 +1,240 @@
+module.exports = {
+  extends: ['eslint-config-prettier', './eslint-test'],
```

Comment:
`eslint-yoshi-base.js` and `eslint-test.js` copy-pasted here  from `@wix/eslint-config-yoshi-base` (cause it is not public package, and Travis could not get it).